packages/fx-core/src/plugins/resource/bot/v2/bicep.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Inputs, ResourceTemplate } from "@microsoft/teamsfx-api";
+import { Context } from "@microsoft/teamsfx-api/build/v2";
+import * as fs from "fs-extra";
+import path from "path";
+import { BotHostTypes, generateBicepFromFile } from "../../../../common";
+import { ArmTemplateResult } from "../../../../common/armInterface";
+import { Bicep } from "../../../../common/constants";
+import { getTemplatesFolder } from "../../../../folder";
+import { getActivatedV2ResourcePlugins } from "../../../solution/fx-solution/ResourcePluginContainer";
+import { NamedArmResourcePluginAdaptor } from "../../../solution/fx-solution/v2/adaptor";
+import { HostTypes, PluginBot } from "../resources/strings";
+import { BicepConfigs } from "./botSolution";
+import * as utils from "../utils/common";
+
+export function mergeTemplates(templates: ArmTemplateResult[]): ArmTemplateResult {
```

Comment:
It looks like this can be utility method about bicep operation for all xxxSolution.